base/docs/basedocs.jl

```diff
@@ -161,7 +161,7 @@ end
     include expressions, literal values, and symbols. For example:
 
         macro sayhello(name)
-            return :( println("Hello, ", \$name) )
+            return :( println("Hello, ", $name) )
```

Comment:
Thanks for the contribution!

Either you can create a new PR, or force push to this branch (it'll update this PR).